converter/src/test/java/gov/cms/qpp/acceptance/cpc/CpcPlusAcceptanceTest.java

```diff
@@ -21,23 +24,32 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+
 
 class CpcPlusAcceptanceTest {
 
 	private static final Path BASE = Paths.get("src/test/resources/cpc_plus/");
 	private static final Path SUCCESS = BASE.resolve("success");
 	private static final Path FAILURE = BASE.resolve("failure");
+	private static final Path FAILURE_FIXTURE = FAILURE.resolve("fixture.json");
+	private static Map<String, CPCAcceptanceFixture> fixtureValues;
 
 	@BeforeAll
-	static void initMockApmIds() {
+	static void initMockApmIds() throws IOException {
 		ApmEntityIds.setApmDataFile("test_apm_entity_ids.json");
+		TypeReference<Map<String, CPCAcceptanceFixture>> ref =
+				new TypeReference<Map<String, CPCAcceptanceFixture>>() { };
+		fixtureValues = JsonHelper.readJson(FAILURE_FIXTURE, ref);
 	}
 
 	@AfterAll
 	static void resetApmIds() {
 		ApmEntityIds.setApmDataFile(ApmEntityIds.DEFAULT_APM_ENTITY_FILE_NAME);
 	}
 
+	@Disabled
```

Comment:
why are we disabling the success verification?

Reply:
oops, meant to reinstate that